### PR TITLE
docs(architecture): improve `blocs_presentation.dart.md` snippet

### PR DESCRIPTION
**Pull request by lsaudon**
## Status

**READY**

## Breaking Changes

NO

## Description

`FirstBloc, FirstState` is more consistent in the example compared to  `WeatherCubit, WeatherState`

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by tenhobi** [comment]
Hi, how is that more consistent? `WeatherCubit` sound more like what it should do, so it is the right name IMO. What should maybe change are the names of other examples etc., if something more expressing could be used. But I see no issue in this case.

**Comment by lsaudon** [comment]
This was just because WeatherCubit does not exist in the example.
Indeed, maybe do the opposite, find a more concrete name for SecondBloc.

**Comment by lsaudon** [comment]
Maybe better?
- `ClothingBloc`, `ClothingBlocEvent`
- `WeatherBloc`, `RefreshWeather`

File: docs/_snippets/architecture/blocs_presentation.dart.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ class MyWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BlocListener<WeatherCubit, WeatherState>(
+    return BlocListener<WeatherBloc, WeatherState>(
```
Comment (felangel) [inline docs/_snippets/architecture/blocs_presentation.dart.md:7]: If we make this change we will also need to update the corresponding documentation in https://github.com/felangel/bloc/blob/b11fe6a1c00a5cb22fd38b2eba2ae6c0c773cec5/docs/architecture.md?plain=1#L73 (and other translations). The simpler change would be to go back to using `FirstBloc` and `SecondBloc` in order to be consistent with the rest of the docs. Let me know what you think and thanks for the contribution! 🙏

**Review by felangel — CHANGES_REQUESTED** [review]
See my comment and lmk what you think.

**Review by tenhobi — APPROVED** [review]

**Review by felangel — COMMENTED** [review]
LGTM thanks so much for the contribution! 💯

**Pull request merged**